app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/Reservation.js

```diff
@@ -48,3 +52,7 @@ export default function Reservation(props) {
     </OverlayTrigger>
   );
 }
+
+const actions = { showReservationInfoModal: actionsUI.showReservationInfoModal };
```

Comment:
- [ ] It would be better to pass the showReservationInfoModal action as an onClick prop from AvailabilityTimeline as is done with ReservationSlot onClick prop. Then there would be no need to connect this component.